File: app/src/main/java/com.github.anrimian.musicplayer/di/app/library/albums/items/AlbumItemsComponent.java
package com.github.anrimian.musicplayer.di.app.library.albums.items;

import com.github.anrimian.musicplayer.ui.library.albums.items.AlbumItemsPresenter;

import dagger.Subcomponent;

@Subcomponent(modules = AlbumItemsModule.class)
public interface AlbumItemsComponent {

    AlbumItemsPresenter albumItemsPresenter();
}

File: app/src/main/java/com/github/anrimian/musicplayer/di/app/library/albums/items/AlbumItemsComponent.kt
package com.github.anrimian.musicplayer.di.app.library.albums.items

import com.github.anrimian.musicplayer.ui.library.albums.items.AlbumItemsPresenter
import dagger.Subcomponent

@Subcomponent(modules = [AlbumItemsModule::class])
interface AlbumItemsComponent {

    fun albumItemsPresenter(): AlbumItemsPresenter

}